Readme.md

```diff
@@ -0,0 +1,940 @@
+  "request_version": "1.0",
+  "pan": "<<PAN_NUMBER>>",
+  "request_medium": "<<web | android | ios>>",
+  "sdk_name": "1",
+  "data": {
+    "transactions": [
+      {
+        "2019-20": {
+          "PersonalInfo": {
+            "Name": "<<NAME>>",
+            "PAN": "<<PAN_NUMBER>>",
+            "DOB": "<<DATE_OF_BIRTH>>",
+            "Address": {
+              "ResidenceName": "",
+              "LocalityOrArea": "<<AREA>>",
+              "RoadOrStreet": "",
+              "ResidenceNo": "<<RESIDENCE_NO>>",
+              "PinCode": "<<PIN>>",
+              "CityOrTownOrDistrict": "<<CITY_OR_TOWN>>",
+              "MobileNo": "91 - <<PHONE>>",
+              "State": "<<STATE>>",
+              "EmailAddress": "<<EMAIL>>"
+            },
+            "AadhaarCardNo": "<<AADHAAR_NUMBER>>",
+            "EmployerCategory": "Not Applicable(eg. Family pension etc)"
+          },
+          "ITR1_IncomeDeductions": {
+            "ProfitsInSalary": "0",
+            "Salary": "<<SALARY>>",
+            "AlwnsNotExempt": "0",
+            "IncomeFromSal": "<<SALARY>>",
+            "UsrDeductUndChapVIA": {
+              "Section80DD": "0\r0",
+              "TotalChapVIADeductions": "0",
+              "Section80GGA": "0",
+              "Section80CCG": "0",
+              "Section80GG": "0",
+              "Section80CCDEmployer": "0",
+              "Section80CCD1B": "0",
+              "Section80GGC": "0",
+              "Section80TTA": "0",
+              "Section80DHealthInsPremium": {
+                "Sec80DHealthInsurancePremiumUsr": "0\r0\r0"
+              },
+              "Section80CCDEmployeeOrSE": "0",
+              "Section80E": "0",
+              "Section80C": "0",
+              "Section80CCC": "0",
+              "Section80EE": "0"
+            },
+            "IncomeOthSrc": "0",
+            "GrossTotIncome": "0",
+            "TotalIncomeOfHP": "0",
+            "TotalIncome": "0",
+            "PerquisitesValue": "0"
+          },
+          "ITR1_TaxComputation": {
+            "TotalTaxPayable": "0",
+            "Rebate87A": "0",
+            "EducationCess": "0",
+            "GrossTaxLiability": "0",
+            "Section89": "0",
+            "NetTaxLiability": "0",
+            "IntrstPay": {
+              "IntrstPayUs234A": "0",
+              "IntrstPayUs234C": "0\r0\r0"
+            },
+            "TotalIntrstPay": "0"
+          },
+          "TaxPaid": {
+            "TaxesPaid": {
+              "TDS": "0\r0\r0\r0",
+              "TotalTaxesPaid": "0"
+            },
+            "BalTaxPayable": "0"
+          },
+          "refund": {
+            "RefundDue": "0",
+            "BankAccountDtls": {
+              "PriBankDetails": {
+                "IFSCCode": "<<IFSC>>",
+                "BankName": "<<BANK_NAME>>",
+                "BankAccountNo": "<<ACCOUNT_NUMBER>>"
+              }
+            }
+          }
+        }
+      },
+      {
+        "2017-18": {
+          "ITR1FORM:ITR1": {
```

Comment:
I think the response format has changes @iAziz786 Can you share the correct format. `ITR1FORM:` is removed

Reply:
@thefoodiecoder Yes, I think it's an old response. @divyank96 can you please copy the latest from zoop-web-sdk.

Reply:
already did and raised the PR again can you please verify once @iAziz786 in develop branch